WordPress/src/androidTest/java/org/wordpress/android/e2e/LoginTests.java

```diff
@@ -0,0 +1,22 @@
+package org.wordpress.android.e2e;
+
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wordpress.android.e2e.pages.MePage;
+import org.wordpress.android.support.BaseTest;
+
+import static org.wordpress.android.BuildConfig.E2E_WP_COM_USER_USERNAME;
+
+@RunWith(AndroidJUnit4.class)
+public class LoginTests extends BaseTest {
+    @Test
+    public void testWPComLoginLogout() {
+        wpLogin();
+        sleep();
+
+        new MePage().go().verifyUsername(E2E_WP_COM_USER_USERNAME);
+        wpLogout();
```

Comment:
Interesting, the only difference is verifying the username. `wpLogout` is 

`
new MePage()
                .go()
                .logout();
`

Reply:
It may have something to do with the presence of the check inside of `verifyUsername`. 

One thing I've found with espresso, is there needs to basically be a `waitForElementToBeDisplayed` call before performing any action on any item – otherwise there can be issues 😢

Reply:
Yep! So I've made this change I have no problem including the verification step instead of another sleep but soon I'll have to make a change to update the LoginFlow to verify the username based on the config. Right now it's verifying based on the WPCOM user